content/en/guides/core/artifacts/track-external-files.md

```diff
@@ -173,16 +180,17 @@ artifact = wandb.Artifact("mnist", type="dataset")
 artifact.add_reference("file:///mount/datasets/mnist/")
 run.log_artifact(artifact)
 ```
+{{% alert color="secondary" %}}
+By default, W&B imposes a 10,000 file limit when adding a reference to a directory. You can adjust this limit by specifying `max_objects=` when you call `add_reference()`.
```

Comment:
It would be good to keep this in sync with the previous admonition with the same details.